bootcd/rpms/genesis_scripts/src/run-genesis-bootloader

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+source /etc/sysconfig/genesis
+
+# send stdout and stderr to the log file
+exec &>${GENESIS_LOGFILE}
+
+echo $$>${GENESIS_PIDFILE}
```

Comment:
You should assume a sane default if genesis_pidfile isn't set in the sysconfig. Same with genesis_logfile